AttackItem.cpp

```diff
@@ -1 +1,11 @@
-#include "AttackItem.h"
\ No newline at end of file
+#include "AttackItem.h"
+
+AttackItem::AttackItem(): Item("Attack Item", 10)
+{
+
+}
+
+void AttackItem::use(Character *)
+{
+    
+}
```

Comment:
FIXME nothing happens when this attack item is used.  

- [ ] Look at the other Item-derived classes, and do what they do in their `use()` function.